src/try-convert/Program.cs

```diff
@@ -32,16 +32,17 @@ private static async Task<int> Main(string[] args)
                 .AddOption(new Option(new[] { "-p", "--project" }, "The path to a project to convert") { Argument = new Argument<string?>(() => null) })
                 .AddOption(new Option(new[] { "-w", "--workspace" }, "The solution or project file to operate on. If a project is not specified, the command will search the current directory for one.") { Argument = new Argument<string?>(() => null) })
                 .AddOption(new Option(new[] { "-m", "--msbuild-path" }, "The path to an MSBuild.exe, if you prefer to use that") { Argument = new Argument<string?>(() => null) })
-                .AddOption(new Option(new[] { "-tfm", "--target-framework" }, "The name of the framework you would like to upgrade to") { Argument = new Argument<string?>(() => null) })
+                .AddOption(new Option(new[] { "-tfm", "--target-framework" }, "The name of the framework you would like to upgrade to. Useful if you'd prefer not to change TFMs to .NET Core just yet.") { Argument = new Argument<string?>(() => null) })
```

Comment:
This kind of obviates the `-tfm` option, I'm just putting this up here now but I wanna think about this one before doing anything else

Reply:
Thought about it more, it's not obviated